bugwarrior/services/gitlab.py

```diff
@@ -314,12 +318,6 @@ def filter_repos(self, repo):
         # fallback if no filter is set
         is_included = True
 
-        if self.include_repos:
-            if repo['path_with_namespace'] in self.include_repos:
-                return True
-            else:
-                is_included = False
-
```

Comment:
I think these lines should not be removed for the circumstance where the `include_repos` and `include_regex` settings are both being used.

Reply:
Good call - I've added them back in with 849494e